src/features/landing/components/AboutVilnaSection/AboutVilnaSecondaryImages/AboutVilnaSecondaryImages.tsx

```diff
@@ -0,0 +1,145 @@
+import { Box, Container } from '@mui/material';
+
+import AboutVilnaSecondaryShape from '../../../../../../public/assets/img/AboutVilnaSecondaryShape.png';
+import useScreenSize from '../../../hooks/useScreenSize/useScreenSize';
+import AboutVilnaBackgroundWithMainSvg from '../AboutVilnaBackgroundWithMainSvg/AboutVilnaBackgroundWithMainSvg';
+import AboutVilnaBackgroundWithSecondaryPng from '../AboutVilnaBackgroundWithSecondarySvg/AboutVilnaBackgroundWithSecondaryPng';
```

Comment:
difficult to read, need change approach about folder, files, class, function, variables names
https://medium.com/@kjschelling/a-comprehensive-guide-to-naming-conventions-in-javascript-46a7518e4807